app/vmalert/utils/links.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import "strings"
+
+const prefix = "/vmalert/"
+
+func Prefix(path string) string {
```

Comment:
Maybe it would be better set prefix as param? and set by default const prefix if it empty

Reply:
Could you please elaborate more on this?